core/chains/evm/gas/rollups/l1_oracle.go

```diff
@@ -59,7 +59,7 @@ func NewL1GasOracle(lggr logger.Logger, ethClient l1OracleClient, chainType chai
 		return nil, nil
 	}
 
-	// TODO implementation to use the clientsByChainID should update the check accordingly, potentially return errors instead of logging
+	// TODO implementation to use the clientsByChainID should update the check accordingly, potentially return errors instead of logging, JIRA https://smartcontract-it.atlassian.net/browse/CCIP-3551
```

Comment:
sorry should've specified earlier, to obfuscate just include it as `// TODO(CCIP-3551) ...`